examples/ex15p.cpp

```diff
@@ -87,6 +87,7 @@ int main(int argc, char *argv[])
    int nc_limit = 3;         // maximum level of hanging nodes
    bool visualization = true;
    bool visit = false;
+   int which_estimator = 0;
```

Comment:
Description at the top of the file has to be changed to include the choice of the estimator